go.mod

```diff
@@ -93,3 +93,5 @@ require (
 )
 
 go 1.19
+
+replace github.com/DataDog/datadog-api-client-go/v2 => /Users/josh.huie/go/src/github.com/datadog-api-client-go
```

Comment:
This line is required for local testing: I will remove it once the spec PRs are merged.